controllers/provisioners/ekscloudformation/ekscloudformation_test.go

```diff
@@ -1272,3 +1272,14 @@ func TestGetNodeAutoScalingGroupMetrics(t *testing.T) {
 	}
 }
 
+func TestParseCustomResourceYaml(t *testing.T) {
+	empty, err := common.ParseCustomResourceYaml("")
+	if (len(empty.Object) > 0)  {
+		t.Fatal("Object should be empty")
+		t.Fail();
```

Comment:
I think `t.Fail()` is not needed here if you already do `t.Fatal()`
Also can you add a more descriptive message (for both `t.Fatal()` calls) that includes `expected: <>, got: <>`